cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -46,7 +46,6 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.ArgumentCaptor;
-
```

Comment:
Why this change? I think we normaly have an empty line between the `java.*` imports and the other imports.

Reply:
Good catch :)